src/frontend/apps/lti_site/types/metadata.ts

```diff
@@ -14,3 +14,11 @@ export interface VideoMetadata extends ResourceMetadata {
     segment_duration_seconds: number;
   };
 }
+
+export interface TimedTextMetadata {
```

Comment:
This is quite the same as `ResourceMetadata`. I think you can do a refactor to not have to redefine them using a generic